libr/cons/cons.c

```diff
@@ -2235,6 +2241,7 @@ R_API void r_cons_thready(void) {
 	r_th_lock_enter (r_cons_lock);
 	r_cons_instance = R_NEW0 (RCons);
 	r_th_lock_leave (r_cons_lock);
+	C->unbreakable = true;
```

Comment:
Shouldn't we be holding the `r_cons_lock` while setting `unbreakable = true`?

Reply:
totally. and this is why i shouldnt merge things when having migraine :D 

thnks for the catch!